packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -625,20 +625,22 @@ class DropdownButton<T> extends StatefulWidget {
   /// if the first item were selected.
   final T value;
 
-  /// Displayed if [value] is null.
+  /// A placeholder widget that is displayed if no item is selected.
```

Comment:
if no item is selected, i.e. if [value] is null.

Reply:
Done